### PR TITLE
Yep 219 Change User Settings

### PR DESCRIPTION
**Pull request by iittooss**
change branch name from change email to change user settings. 

I prefer to make a branch of the pure front and another for the logic knowing that the components are close

### CODE REVIEW

File: front/src/assets/i18n/en.json [hunk 1]
```diff
@@ -89,6 +91,37 @@
       "title": "Invisible mode",
       "on": "Invisible",
       "off": "Visible"
+    },
+    "userAccount": "Change your informations",
```
Comment (ASparton) [inline front/src/assets/i18n/en.json:95]: No s to *information*

**Review by ASparton — APPROVED** [review]

File: front/src/components/Settings/Password.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          onChange={(e) => setValue(e.target.value)}
+        />
+        <PasswordInput
+          error={errorConfirmation}
+          label={t('settings.password.confirmation')}
+          placeholder={t('settings.password.placeholder')}
+          onChange={(e) => setConfirmation(e.target.value)}
+        />
+      </SettingsDrawer>
+    </>
```
Comment (filveith) [inline front/src/components/Settings/Password.tsx:93]: No working

Comment (filveith) [reply front/src/components/Settings/Password.tsx:93]: Hehe in the end it works

**Review by filveith — APPROVED** [review]

File: front/src/assets/i18n/en.json (hunk 1 above)
Comment (filveith) [reply front/src/assets/i18n/en.json:95]: Chut

Comment (filveith) [inline front/src/assets/i18n/en.json:95]:
```suggestion
    "userAccount": "Change your information",
```

**Review by filveith — APPROVED** [review]

**Review by ASparton — APPROVED** [review]

**Pull request merged**